app/position/tracking/positiontrackingmanager.cpp

```diff
@@ -332,7 +332,7 @@ void PositionTrackingManager::setVariablesManager( VariablesManager *newVariable
 
 QgsCoordinateReferenceSystem PositionTrackingManager::crs() const
 {
-  return QgsCoordinateReferenceSystem::fromEpsgId( 4326 );
+  return PositionKit::positionCRS();
```

Comment:
could you please verify what CRS is used in the QGIS plugin when creating the tracking layer? If we're assigning 2D CRS to it upon creation, we should change that to 3D CRS...